README.md

```diff
@@ -1,167 +1,45 @@
+2. Clone your new repository
+3. Edit Chart.yaml:
+    - name:        <application_name>
+    - description: <application_description>
+    - appVersion:  <application_version>
+    - home:        <application_repository_URL>
+    
+4. Update helm dependencies:  helm dependency update .
 
-### Copy and Clone
-
-This repo is a [GitHub Template Repository](https://help.github.com/en/articles/creating-a-repository-from-a-template). You can create a private copy by using the "Use this Template" link in the repo:
-
-![Template Repo](https://docs.github.com/assets/cb-95207/mw-1000/images/help/repository/use-this-template-button.webp)
-
-You should use the template to create a new **private** repo in your org, for example `mycompany/replicated-helm` or `mycompany/replicated-starter-helm`.
-
-Once you've created a repository from the template, you'll want to `git clone` your new repo and `cd` into it locally.
-
-
-
-### 1. Install CLI
-
-To start, you'll want to install the `replicated` CLI.
-You can install with [homebrew](https://brew.sh) or grab the latest Linux or macOS version from [the replicatedhq/replicated releases page](https://github.com/replicatedhq/replicated/releases).
-
-#### Brew
-
-```shell script
-brew install replicatedhq/replicated/cli
-```
-
-#### Manual
-
-```shell script
-curl -s https://api.github.com/repos/replicatedhq/replicated/releases/latest \
-           | grep "browser_download_url.*$(uname | tr '[:upper:]' '[:lower:]')_amd64.tar.gz" \
-           | cut -d : -f 2,3 \
-           | tr -d \" \
-           | cat <( echo -n "url") - \
-           | curl -fsSL -K- \
-           | tar xvz replicated
-```
-Then move `./replicated` to somewhere in your `PATH`:
-
-
-```shell script
-mv replicated /usr/local/bin/
-```
-
-#### Verifying
-
-You can verify it's installed with `replicated version`:
-
-```text
-$ replicated version
-```
-```json
-{
-  "version": "0.31.0",
-  "git": "c67210a",
-  "buildTime": "2020-09-03T18:31:11Z",
-  "go": {
-      "version": "go1.14.7",
-      "compiler": "gc",
-      "os": "darwin",
-      "arch": "amd64"
-  }
-}
-```
-
-
-### Configure environment
-
-You'll need to set up two environment variables to interact with vendor.replicated.com:
-
-```
-export REPLICATED_APP=...
-export REPLICATED_API_TOKEN=...
-```
-
-`REPLICATED_APP` should be set to the app slug from the Settings page:
-
-<p align="center"><img src="./doc/REPLICATED_APP.png" width=600></img></p>
+5. Install Replicated CLI - [Instructions](https://docs.replicated.com/reference/replicated-cli-installing#install-the-replicated-cli)
 
-Next, create a Service Account API token from the vendor portal under [Service Accounts](https://vendor.replicated.com/team/serviceaccounts):
+6. Configure environment variables -- [Instructions](https://docs.replicated.com/reference/replicated-cli-installing#set-environment-variables)
```

Comment:
Part of me says move this down to a 'releasing' section. Let people get started with helm w/o these steps since they won't use them until they have done local testing. In fact, it might make since to never even tell them to do this and just tell them to enable the github actions once they are ready to start pushing to their development channel.

If they don't have to go to Vendor Portal at all to get started, maybe they'll get started and get to their first chart test cycle faster. That's what I'm looking to optimize here, get them through a "helm template, change something, helm template" cycle as soon as possible.